packages/skatejs/src/with-update.js

```diff
@@ -159,7 +159,9 @@ export const withUpdate = (Base: Class<any> = HTMLElement): Class<any> =>
       // only once when the custom element is defined. If we did this only in
       // the constructor, then props would not link to attributes.
       defineProps(this);
-      return this._observedAttributes;
+      return Array.from(
```

Comment:
IE doesn't support `Array.from()` and doesn't support an iterable being passed to `Set`.

Reply:
You should just be able to write this as: `return this._observedAttributes.concat(super.observedAttributes)`, or similar.